gradlew

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
+#!/bin/sh
 
 #
-# Copyright 2015 the original author or authors.
+# Copyright © 2015-2021 the original authors.
```

Comment:
```suggestion
# Copyright © 2015-2023 the original authors.
```

Reply:
I wish I knew why the wrapper script does not have the latest year in the copyright, and of course I always forget to check. Thanks @david-limbach!